Core/CategoryDTO.cs

```diff
@@ -1,25 +1,16 @@
-﻿namespace Core
+﻿using System.Collections.Generic;
+
+namespace Core
 {
     public class CategoryDto
     {
-		public CategoryDto(string name, string cardNumber, string applicationCode, string isIncome, 
-			string isExpence, string terminal, string description)
+		public CategoryDto(string name, List<Rule> rules)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this constructor to use a generic collection designed for inheritance.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4970545)